### ISSUE TITLE
Allow the configuration of multiple LINSTOR Controllers

### ISSUE THREAD

**Issue by dvance** [beornf/linstor-docker-volume#19]
In my environment I configured a highly-available LINSTOR controller as per the instructions here: https://linbit.com/drbd-user-guide/linstor-guide-1_0-en/#s-linstor_ha

I had attempted to configure this plugin with multiple, comma-separated, LINSTOR Controllers like so:
```
[global]
storagepool = lvm-thin 
fs = xfs 
size = 100MB
replicas = 2
controllers = linstor://192.168.221.60,linstor://192.168.221.61,linstor://192.168.221.62
```
This would result in the below error unless the controller was running on the first listed address (192.168.221.60).
``` 
root@linstor-0:~# docker volume create --opt fs=xfs --opt size=1G -d linbit/linstor-docker-volume test03
Error response from daemon: create test03: VolumeDriver.Create: Post "http://192.168.221.60:3370/v1/resource-definitions": dial tcp 192.168.221.60:3370: connect: connection refused
```
I was able to work around the issue fairly easily, by configuring drbd-reactor to use a virtual IP address and just set that within the docker-volume plugin configuration. Still, it would be nice if this wasn't necessary and the docker-volume plugin could "round-robin" attempts to different controllers much like the linstor-client can be configured to do.

Consider this a feature request. Thanks a bunch for the work on this plugin!

**Comment by rck** [comment]
I think this could be rather simple as the plugin uses `golinstor` and it gained "multi client support" some time ago: https://pkg.go.dev/github.com/LINBIT/golinstor/client#example-package-MultipleControllers

**Comment by beornf** [comment]
@rck Yes should be a simple fix. Do you think I should make changes to this PR https://github.com/beornf/linstor-docker-volume/pull/17 or start a new one?

**Comment by rck** [comment]
guess just go for a new one...
